linux/Dockerfile

```diff
@@ -21,13 +22,28 @@ ENV DEBCONF_NOWARNINGS yes
 RUN apt-get -q -y update &&  \
   apt-get -q -y install devscripts equivs meson python3 python3-setuptools software-properties-common && \
   add-apt-repository ppa:keymanapp/keyman && \
-  add-apt-repository ppa:keymanapp/keyman-alpha && \
+  add-apt-repository ppa:keymanapp/keyman-alpha
+RUN apt-get -q -y update &&  \
   apt-get -q -y upgrade
 
+# Install dependencies
 ADD debian/control /tmp/control
 # Answer 'yes' to install questions
 RUN (yes | mk-build-deps --install /tmp/control) || true
+
+# Install Node
 RUN curl -sL https://deb.nodesource.com/setup_18.x | bash
 RUN apt-get -q -y install nodejs
+
+# Install emscripten
+RUN cd /usr/share && \
+  git clone https://github.com/emscripten-core/emsdk.git && \
+  cd emsdk && \
+  ./emsdk install latest && \
+  ./emsdk activate latest && \
+  echo 'source "/usr/share/emsdk/emsdk_env.sh"' >> $HOME/.bashrc && \
+  echo "export EMSDK_NODE=/usr/bin/node" >> $HOME/.bashrc && \
```

Comment:
Again, we specifically don't want emscripten on the path -- use EMSCRIPTEN_BASE to specify the path to emcc etc